src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.Arrays;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Bulldozer bulldozer = new Bulldozer();
+    private Bulldozer bulldozer1 = new Bulldozer();
+    private Bulldozer bulldozer2 = new Bulldozer();
+
+    public List<Bulldozer> get() {
```

Comment:
Consider creating a local variable in your implementation of `get()` method to store the result before returning it. This can make debugging easier and your code more readable.